src/components/AppProvider/AppProvider.scss

```diff
@@ -32,7 +32,7 @@ button {
 
 html {
   position: relative;
-  font-size: ($base-font-size / $default-browser-font-size) * 100%;
+  font-size: 100%;
```

Comment:
Is this the right time to introduce this change? It does impact the rem value we would set the `font-size` and `line-height` tokens to.

For context, there has been a previous attempt to make this change, but it was reverted https://github.com/Shopify/polaris-react/pull/1590

Reply:
Might be worth getting @martenbjork to weigh in on this. I know he's experimented with global font size changes in the past

Reply:
@alex-page ok probably best not to conflate things. it's simple enough to update the token values when we do make the change.